src/lib.rs

```diff
@@ -388,7 +388,13 @@ fn create_settings(matches: &ArgMatches) -> Settings {
         std::process::exit(1)
     });
 
-    settings.argv = argv;
+    settings.argv = argv.clone();
+    settings.orig_argv = vec![env::current_exe()
+        .unwrap_or_default()
+        .to_str()
+        .unwrap()
+        .to_string()];
+    settings.orig_argv.append(&mut argv);
```

Comment:
This is post-processed by clap. That doesn't look like 'orig' enough. Using [std::env::args](https://doc.rust-lang.org/std/env/fn.args.html) will fit better.